docs/user-guide/members/README.md

```diff
@@ -8,16 +8,62 @@ keywords: members, project participants
-| [Add members](#add-members)                     | How to add existing members or invite new members to a project. |
-| [Edit members](#edit-members)                   | How to change the role of a member in a project.                |
-| [Remove members](#remove-members)               | How to remove members from a project.                           |
-| [Roles and permissions](#roles-and-permissions) | How to manage roles and permissions for members.                |
-| [Groups](#groups)                               | How to add members to a group and add groups to a project.      |
+| Topic                                                 | Content                                                      |
+| ----------------------------------------------------- | ------------------------------------------------------------ |
+| [Project members overview](#project-members-overview) | How to get an overview of all project members.               |
+| [Add members](#add-members)                           | How to add existing members or invite new members to a project. |
+| [Edit members](#edit-members)                         | How to change the role of a member in a project.             |
+| [Remove members](#remove-members)                     | How to remove members from a project.                        |
+| [Roles and permissions](#roles-and-permissions)       | How to manage roles and permissions for members.             |
+| [Groups](#groups)                                     | How to add members to a group and add groups to a project.   |
 
 <video src="https://openproject-docs.s3.eu-central-1.amazonaws.com/videos/OpenProject-Invite-and-Manage-Members.mp4" type="video/mp4" controls="" style="width:100%"></video>
 
+## Project members overview
+
+On the left side menu you will see **Members**. When selected, it will show a list of project members, project groups, as well as the users, with whom work packages from this project have been shared. You can **edit** or **delete** a user or a group by click the respective icon at the end of the line listing said user or group.
+
+![Project members overview in OpenProject](members-overview.png)
+
+Standard filters on the left side menu include the following: 
+
+- **All** - returns all members and groups of the project, as well as non-members, with whom a work package from this project has been shared
+
+- **Locked** - returns all locked users that are members of this project, as well as locked non-members, with whom a work package from this project has been shared
+
+- **Invited** - returns all users that have been invited, but have not yet registered
+
+- **Project roles** provides filters based on all the user roles that have been assigned to users in that specific project. 
+
+- **Work package shares** provides filters based on all the roles available for sharing work packages. They include
```

Comment:
I would do two dots at the end of this sentence and indent one level bellow all the possible options (All shares, view, comment and edit). Like this:

... They include:
   - All shares
   - View
   - Comment
   - Edit